cinema/serializers.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import serializers
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(required=True, max_length=100)
+    description = serializers.CharField(required=False, max_length=100)
```

Comment:
The `description` field has a `max_length` of 100 in the serializer, but the `Movie` model's migration file specifies a `max_length` of 255. Update the serializer to have a `max_length` of 255 to match the model and ensure consistency.